apps/fluent-tester/src/TestComponents/Switch/E2ESwitchTest.tsx

```diff
@@ -22,7 +23,7 @@ export const E2ESwitchTest: React.FunctionComponent = () => {
   );
 
   return (
-    <Stack style={stackStyle}>
+    <Stack style={Platform.OS === 'android' ? { ...commonTestStyles.androidContainer, height: 100 } : stackStyle}>
```

Comment:
Please create these style objects as memoized objects and pass them in instead of passing in an object literal into the style prop to avoid extra rerenders.

Same comment for the other test style changes

Reply:
Changes done and approved.